### PR TITLE
TMA-602: User brick can't add user into existing user groups

### PR DESCRIPTION
**Pull request by vhtien**

### CODE REVIEW

File: lib/gooddata/lcm/actions/synchronize_users.rb [hunk 1]
```diff
@@ -96,6 +96,7 @@ def call(params)
           ignore_failures = GoodData::Helpers.to_boolean(params.ignore_failures)
           remove_users_from_project = GoodData::Helpers.to_boolean(params.remove_users_from_project)
           do_not_touch_users_that_are_not_mentioned = GoodData::Helpers.to_boolean(params.do_not_touch_users_that_are_not_mentioned)
+          create_non_existing_user_groups = GoodData::Helpers.to_boolean(params.create_non_existing_user_groups || true)
```
Comment (yenkins) [inline lib/gooddata/lcm/actions/synchronize_users.rb]: Refers to 'params' more than self (maybe move it to another class?) (FeatureEnvy)

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/297/ : SUCCESS in 6s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/15/ : SUCCESS in 1m 53s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/326/ : SUCCESS in 2m 16s

File: lib/gooddata/lcm/actions/synchronize_users.rb [hunk 2]
```diff
@@ -111,6 +112,8 @@ def call(params)
           end
 
           data.each do |row|
+            params.gdc_logger.debug("Processing row: #{row}")
```
Comment (yenkins) [inline lib/gooddata/lcm/actions/synchronize_users.rb]: Calls 'params.gdc_logger' 2 times (DuplicateMethodCall)

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/299/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/17/ : SUCCESS in 48s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/328/ : SUCCESS in 36s

**Comment by vhtien** [comment]
integration test: https://ms-gerrit.intgdc.com/#/c/1092/

**Comment by panjan** [comment]
ok to merge

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-merge-compliance-check/109/ : SUCCESS in 12s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/20/ : SUCCESS in 42s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/331/ : SUCCESS in 38s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/962/ : SUCCESS in 44m 42s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/725/ : SUCCESS in 22m 07s

**Pull request merged**